content/hi/docs/concepts/architecture/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "क्लस्टर आर्किटेक्चर"
+weight: 30
+description: >
+  The architectural concepts behind Kubernetes.
+---
+
+{{< figure src="/images/docs/kubernetes-cluster-architecture.svg" alt="Components of Kubernetes" caption="Kubernetes क्लस्टर आर्किटेक्चर" class="diagram-large" >}}
```

Comment:
It will be good if alt message should also be in hindi.

optionally: image having content should also be in localize language